website/docs/r/cloud_service_epg.html.markdown

```diff
@@ -0,0 +1,92 @@
+---
+subcategory: - "Cloud"
+layout: "aci"
+page_title: "ACI: aci_cloud_service_epg"
+sidebar_current: "docs-aci-resource-cloud_service_epg"
+description: |-
+  Manages ACI Cloud Service EPG
+---
+
+# aci_cloud_service_epg #
+
+Manages ACI Cloud Service EPG
+
+## API Information ##
+
+* `Class` - cloudSvcEPg
+* `Distinguished Name` - uni/tn-{name}/cloudapp-{name}/cloudsvcepg-{name}
+
+## GUI Information ##
+
+* `Location` - Application Management -> EPGs
+
+
+## Example Usage ##
+
+```hcl
+resource "aci_cloud_service_epg" "example" {
+  cloud_applicationcontainer_dn  = aci_cloud_applicationcontainer.example.id
+  name                           = "example"
+  access_type                    = "Public"
+  annotation                     = "orchestrator:terraform"
+  deployment_type                = "CloudNative"
+  flood_on_encap                 = "disabled"
+  label_match_criteria           = "AtleastOne"
+  cloud_service_epg_type         = "Azure-SqlServer"
+  cloud_rs_cloud_epg_ctx         = aci_vrf.example.id
+}
+```
+
+## Argument Reference ##
+
+* `cloud_applicationcontainer_dn` - (Required) Distinguished name of the parent Cloud Application container object.
+* `name` - (Required) Name of the Cloud Service EPG object.
+* `annotation` - (Optional) Annotation of the Cloud Service EPg object.
+* `name_alias` - (Optional) Name Alias of the Cloud Service EPg object.
+* `access_type` - (Optional) cloud service access type.accessType refers to the type of connectivity to the service. It could be a public or private connectivity. Allowed values are "Private", "Public", "PublicAndPrivate", "Unknown", and default value is "Public". Type: String.
```

Comment:
* `access_type` - (Optional) This refers to the type of connectivity to the service. It could be a public or private connectivity. Allowed values are "Private", "Public", "PublicAndPrivate", "Unknown", and default value is "Public". Type: String.